oarepo_oidc_einfra/resources.py

```diff
@@ -67,18 +203,40 @@ def upload_dump(self):
                 "message": "Content-Type must be application/json",
             }, 400
 
-        dump_url = current_app.config["EINFRA_DUMP_DATA_URL"]
-        now = datetime.utcnow().strftime("%Y-%m-%d-%H-%M-%S")
-        dump_path = f"{dump_url}/{now}.json"
+        dump_path, checksum = store_dump(request.data)
+        update_from_perun_dump.delay(dump_path, checksum)
```

Comment:
Check permission on current identity to trigger update

Reply:
The check is few lines above:

```
        if not Permission(upload_dump_action).allows(g.identity):
            raise PermissionDeniedError()
```